roles/ora-host/tasks/validate_passwords.yml

```diff
@@ -56,7 +56,7 @@
 
 - name: Fail if gcloud command failed
   fail:
-    msg: "Command failed: stderr={{ sys_pass_result.stderr }}"
+    msg: "Command failed with rc={{ sys_pass_result.rc }}: stderr={{ sys_pass_result.stderr }}"
```

Comment:
Can you make the same update to https://github.com/google/oracle-toolkit/blob/44605a7786c359d41bb236a717cdec6458844899/roles/workload-agent/tasks/create_db_user.yml#L26 too?